pre-lib-harddrive.sh

```diff
@@ -0,0 +1,56 @@
+# Common functions for %pre kickstart section of hard drive tests.
+
+# Download the ISO on a hard drive.
+function prepare_iso() {
+    local url="$1"
+    local disk="$2"
+
+    wipefs -a "${disk}"
+    mkfs.ext4 -F "${disk}"
+    mkdir /prep-mount
+    pushd /prep-mount
```

Comment:
I don't think it's a good idea to create folder in `/`. Could we please instead use `/tmp/`?

Reply:
Isn't /tmp tmpfs ? Perhaps /var/tmp ?